lib/generators/geocoder/config/templates/initializer.rb

```diff
@@ -10,6 +10,7 @@
   # api_key: nil,               # API key for geocoding service
   # cache: nil,                 # cache object (must respond to #[], #[]=, and #del)
   # cache_prefix: 'geocoder:',  # prefix (string) to use for all cache keys
+  # cache_expiration: 1.day,    # expiration time (ttl)
```

Comment:
We should probably create only `cache_options` here (no _prefix or _expiration).

Reply:
@alexreisner I see I forgot to remove `cache_expiration` from docs, for `cache_prefix` it's for backward compatibility, please check my comment here https://github.com/alexreisner/geocoder/pull/1532#discussion_r745106020 and let me know